entrypoint.sh

```diff
@@ -109,8 +109,19 @@ function appsodyStart() {
 		cmd=debug
 	fi
 
-	if [ -f "env.properties" ]; then
-		dopts=--docker-options="--env-file=env.properties"
+	ENV_PROPERTIES_FILE=env.properties
+	PROJECT_LINKS_ENV_FILE="$ROOT/.codewind-project-links.env"
+	dopts=""
+	if [ -f "$ENV_PROPERTIES_FILE" ] && [ -f "$PROJECT_LINKS_ENV_FILE" ]; then
+		# if both files exist, merge them to resolve problem with docker-options
+		TEMP_ENV_FILE="/temp_env_file.env"
```

Comment:
Maybe we should put this in the project as well rather than `/`?  There's also danger of collision if we used a common path

Reply:
Will change to `TEMP_ENV_FILE=".codewind-merged-env-files.env"` so its in the project dir and has a `.codewind` prefix